fw/fs.c

```diff
@@ -15,8 +15,8 @@ static uint8_t fs_force_capacity = 0;
 static int32_t file_startblock_idx;
 static uint32_t file_byte_idx;
 static uint32_t file_byte_len;
-static uint32_t file_seq;
 static int32_t file_prev_skip_startblock_idx;
+uint32_t g_fs_file_seq;
```

Comment:
add comment to say that this is only up to date when fs_info() or fs_open() is called